pyproject.toml

```diff
@@ -1,6 +1,62 @@
 [build-system]
-requires = [ "setuptools>=62", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = [ "poetry-core>=2.0.0,<3.0.0", "wheel"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+packages = [{include = "confluent_kafka", from = "src"}]
+
+[tool.poetry.dependencies]
```

Comment:
We are moving optional dependencies to required ones. Most of the users of this client don't use schema registry and we shouldn't install those dependencies by default.